ocp4/profiles/coreos-ncp.profile

```diff
@@ -86,37 +79,37 @@ selections:
     #- sshd_rekey_limit
 
     # Time Server
-    # - chronyd_client_only
-    # - chronyd_no_chronyc_network
+    - chronyd_client_only
+    - chronyd_no_chronyc_network
 
     ### Network Settings
-    #- sysctl_net_ipv6_conf_all_accept_ra
-    #- sysctl_net_ipv6_conf_default_accept_ra
-    #- sysctl_net_ipv4_conf_all_accept_redirects
-    #- sysctl_net_ipv4_conf_default_accept_redirects
-    #- sysctl_net_ipv6_conf_all_accept_redirects
-    #- sysctl_net_ipv6_conf_default_accept_redirects
-    #- sysctl_net_ipv4_conf_all_accept_source_route
-    #- sysctl_net_ipv4_conf_default_accept_source_route
-    #- sysctl_net_ipv6_conf_all_accept_source_route
-    #- sysctl_net_ipv6_conf_default_accept_source_route
-    #- sysctl_net_ipv4_conf_all_secure_redirects
-    #- sysctl_net_ipv4_conf_default_secure_redirects
-    #- sysctl_net_ipv4_conf_all_send_redirects
-    #- sysctl_net_ipv4_conf_default_send_redirects
-    #- sysctl_net_ipv4_conf_all_log_martians
-    #- sysctl_net_ipv4_conf_default_log_martians
-    #- sysctl_net_ipv4_conf_all_rp_filter
-    #- sysctl_net_ipv4_conf_default_rp_filter
-    #- sysctl_net_ipv4_icmp_ignore_bogus_error_responses
-    #- sysctl_net_ipv4_icmp_echo_ignore_broadcasts
-    #- sysctl_net_ipv4_ip_forward
-    #- sysctl_net_ipv4_tcp_syncookies
+    - sysctl_net_ipv6_conf_all_accept_ra
+    - sysctl_net_ipv6_conf_default_accept_ra
+    - sysctl_net_ipv4_conf_all_accept_redirects
+    - sysctl_net_ipv4_conf_default_accept_redirects
+    - sysctl_net_ipv6_conf_all_accept_redirects
+    - sysctl_net_ipv6_conf_default_accept_redirects
+    - sysctl_net_ipv4_conf_all_accept_source_route
+    - sysctl_net_ipv4_conf_default_accept_source_route
+    - sysctl_net_ipv6_conf_all_accept_source_route
+    - sysctl_net_ipv6_conf_default_accept_source_route
+    - sysctl_net_ipv4_conf_all_secure_redirects
+    - sysctl_net_ipv4_conf_default_secure_redirects
+    - sysctl_net_ipv4_conf_all_send_redirects
+    - sysctl_net_ipv4_conf_default_send_redirects
+    - sysctl_net_ipv4_conf_all_log_martians
+    - sysctl_net_ipv4_conf_default_log_martians
+    - sysctl_net_ipv4_conf_all_rp_filter
+    - sysctl_net_ipv4_conf_default_rp_filter
+    - sysctl_net_ipv4_icmp_ignore_bogus_error_responses
+    - sysctl_net_ipv4_icmp_echo_ignore_broadcasts
+    - sysctl_net_ipv4_ip_forward
```

Comment:
Are you sure it is safe to disable forwarding? All the pods in OCP are NAT-ed, right?

Reply:
This won't work with OpenShift, the kubelet and kubeproxy won't work without ip forwarding enabled.

Reply:
Great! Good feedback on this control. Will make a comment on it.